ci/run_test_post_always.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This is a post test processing script for post processing the
+# run_test.sh stage CI run
+
+set -uex
+
+# shellcheck source ./.build_vars.sh
```

Comment:
(lint) Couldn't parse this shellcheck annotation. [SC1073]
(lint) Expected "disable". Fix any mentioned problems and try again. [SC1072]